src/test/ui/try-block/try-block-maybe-bad-lifetime.stderr

```diff
@@ -21,6 +21,8 @@ LL |             ::std::mem::drop(x);
 LL |         };
 LL |         println!("{}", x);
    |                        ^ value borrowed here after move
+   |
+   = note: this error originates in the macro `$crate::format_args_nl` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
This new output seems non-ideal, but there's probably not a good way to hide it. I guess it could potentially be useful if/when `-Z macro-backtrace` is stabilized?

Reply:
Not great but there are other instances like this.